src/testing/examples/dynamics3d_single_body_test.argos

```diff
@@ -64,17 +64,27 @@
   <!-- * Visualization * -->
   <!-- ****************** -->
   <visualization>
-    <qt-opengl lua_editor="false">
+    <qt-opengl lua_editor="false" show_boundary="false">
       <camera>
-        <placement idx="0" position=" 0,   -1.25,0.5" look_at="0,0,0.05" lens_focal_length="78" />
-        <placement idx="1" position=" 1.25,-1.25,0.6" look_at="0,0,0.05" lens_focal_length="92" />
-        <placement idx="2" position=" 1.25, 0,   0.5" look_at="0,0,0.05" lens_focal_length="78" />
-        <placement idx="3" position=" 1.25, 1.25,0.6" look_at="0,0,0.05" lens_focal_length="92" />
-        <placement idx="4" position=" 0,    1.25,0.5" look_at="0,0,0.05" lens_focal_length="78" />
-        <placement idx="5" position="-1.25, 1.25,0.6" look_at="0,0,0.05" lens_focal_length="92" />
-        <placement idx="6" position="-1.25, 0,   0.5" look_at="0,0,0.05" lens_focal_length="78" />
-        <placement idx="7" position="-1.25,-1.25,0.6" look_at="0,0,0.05" lens_focal_length="92" />
-        <placement idx="8" position="0,0,0.5" look_at="0.0,0.0,0.125" up="1,0,0" lens_focal_length="40"/>
+        <placements>
+          <placement index="0" position=" 0,   -1.25,0.5" look_at="0,0,0.05" lens_focal_length="78" />
```

Comment:
Example looks good--this should be used to update the results of "argos3 -q opengl" with the new functionality

Reply:
I will add this to the to-do list

Reply:
Fixed in e3b31e21f310eac24ca0210a066f1fd9e73982c6